### PR TITLE
Addded code snippets in all supported languages

### PR DESCRIPTION
**Pull request by dcoplowe**
# Description

Added code snippets. 

* Include a summary of the new content you're adding or the reason for the change. 
* Include relevant context.
* Do not link to work items.
* Any PNG images should be compressed using [Tiny PNG](https://tinypng.com/) or equivalent.

## Type of change

Please delete options that are not relevant.

- [x] New document(s)/updating existing
- [ ] Fixes
- [ ] Styling
- [ ] Bug fix (non-breaking change which fixes an issue)

## Reviews and merging

You are responsible for getting your PR merged. Address review comments promptly and **make sure to merge the PR when ready**.
Feel free to 'Enable automerge' - your PR will automatically merge when accepted and passing the build.

### CODE REVIEW

File: docs/core-concepts/connections.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ Connections can have one of the statuses described in the following table.
 
 ## Linked data connection example
 
-Verify that a connection returns a Linked `status` using one of our [Connections](/codat-api#/operations/list-connections) endpoints.
```
Comment (mcclowes) [inline docs/core-concepts/connections.md:37]: We should put in a redirect for this

File: docs/core-concepts/connections.md [hunk 2]
```diff
@@ -115,11 +224,127 @@ PATCH /companies/COMPANY_ID/connections/CONNECTION_ID
...
 ## Delete a data connection
 
-To delete a data connection, send a request to the <a href="/codat-api#/operations/delete-connection" target="_blank">DELETE /connectionId endpoint</a> and provide the `companyId` and `connectionId` as path parameters.
+To delete a data connection, send a request to the <a href="/platform-api#/operations/delete-connection" target="_blank">DELETE /connectionId endpoint</a> and provide the `companyId` and `connectionId` as path parameters.
 
-```http
-DELETE /companies/COMPANY_ID/connections/CONNECTION_ID
-```
+<Tabs>
+  <TabItem value="curl" label="Unix Bash">
```
[the remark this answers is not in the document]
Comment (dcoplowe) [reply docs/core-concepts/connections.md:230]: You're right...

File: docs/core-concepts/connections.md (hunk 1 above)
Comment (dcoplowe) [reply docs/core-concepts/connections.md:37]: Can you expand?

File: docs/using-the-api/querying.md [hunk 3]
```diff
@@ -357,25 +739,111 @@ Query: `dataConnections.status=PendingAuth`
...
+```
+</TabItem>
+<TabItem value="http" label="HTTP">
+
+```http
+GET /companies?query=dataConnections.status=PendingAuth
 ```
 </TabItem>
-</Tabs>  
+</Tabs>
```
Comment (mcclowes) [inline docs/using-the-api/querying.md:847]:
```suggestion
  
  
```

**Review by mcclowes — APPROVED** [review]

**Pull request merged**